packages/theme-dark-extension/style/variables.css

```diff
@@ -120,10 +120,10 @@ all of MD as it is not optimized for dense, information rich UIs.
    */
 
   /* Defaults use Material Design specification */
-  --jp-ui-font-color0: white;
-  --jp-ui-font-color1: var(--md-grey-300);
-  --jp-ui-font-color2: var(--md-grey-500);
-  --jp-ui-font-color3: var(--md-grey-700);
+  --jp-ui-font-color0: rgba(255, 255, 255, 1);
```

Comment:
Any reason we are switching away from these colors to using transparency? I realize material design recommends using transparency like this, but I think @cameronoelsen did a bunch of tests and found it wasn't great in lab. Don't remember the details, but would love to understand the motivation. If we are changing them, should we also change them in the light theme as well?

Reply:
In this case, I'm making the dark theme consistent with the light theme: https://github.com/jupyterlab/jupyterlab/blob/master/packages/theme-light-extension/style/variables.css#L121-L124

Reply:
Sounds good!